src/styles/typography-extends.scss

```diff
@@ -0,0 +1,52 @@
+@import 'styles/settings';
+
+// Big titles, HE-Meter page
+%displa1 {
```

Comment:
Isn't it a typo?

Reply:
Well it is, is lacking a `y`
![image](https://user-images.githubusercontent.com/6906348/59911140-8baa6400-9413-11e9-9196-c36df93b2c86.png)
Will fix it.